internal/uvm/scsi.go

```diff
@@ -303,7 +312,7 @@ func (uvm *UtilityVM) addSCSIActual(ctx context.Context, hostPath, uvmPath, atta
 				MountPath:  sm.UVMPath,
 				Lun:        uint8(sm.LUN),
 				Controller: uint8(sm.Controller),
-				ReadOnly:   readOnly,
+				Options:    options,
```

Comment:
This seems like a breaking change in the communication boundary between shim and gcs. I think we should keep `ReadOnly` on the type, and pass it as well, or rev the version somehow.

Reply:
Completely didn't think about this, thank you for pointing it out! I'll add ReadOnly back in :)